types/react-native/index.d.ts

```diff
@@ -4112,6 +4119,8 @@ export interface VirtualizedListProps<ItemT> extends ScrollViewProps {
 
     viewabilityConfig?: ViewabilityConfig;
 
+    viewabilityConfigCallbackPairs?: ViewabilityConfigCallbackPairs;
```

Comment:
Any reason you just didn't use `ViewabilityConfigCallbackPairs[]`, instead of making a ViewabilityConfigCallbackPairs type?